Assets/Scripts/CombatScripts/BulletShellController.cs

```diff
@@ -20,21 +20,34 @@ void Start()
     private void Update(){
 
         if(firePos != null && (Vector3.Distance(gameObject.transform.position, firePos) > 17f))
-            Destroy(bulletPrefab);
+            Destroy(gameObject);
     }
 
     private void Awake(){
         firePos = gameObject.transform.position;
         
         Rigidbody2D bulletRB = gameObject.GetComponent<Rigidbody2D>();
-        bulletRB.velocity = new Vector2(15f, 0);
+        GameObject player = GameObject.Find("Player");
+        PlayerController playerController = player.GetComponent<PlayerController>();
+        if (playerController.playerDirection == PlayerController.Direction.right)
+        {
+			bulletRB.velocity = bulletTrajectory;
+		}
+        else
+        {
+            bulletRB.velocity = -bulletTrajectory;
+        }
     }
 
     private void OnTriggerEnter2D (Collider2D other){
         // Create Interface for the enemies that each different kind of enemy will implement. Change BasicAIScript to search
         //      for said interface(Down the line, not needed right now)
         if(other.gameObject.layer == 6){
-            Destroy(bulletPrefab);
+            Destroy(gameObject);
+            return;
+        }
+        else if(other.gameObject.layer == 0){

```

Comment:
Camera layer is also 0, doesn't seem like it's created problems but it's probably better practice to make a separate layer just for the player. Thoughts?

Reply:
This type of thing won't change much, but like the previous comment, down the line, we will give the player their own layer. The case I can think of that will be important is that if a bullet was shot by an enemy we don't want it to collide w other enemies.
Somehow, we are going to have to restructure the entire bullet prefab to have certain properties based on who shot the bullet, the direction they were facing and perhaps what kind of bullet.

Reply:
Gonna update in #5